app/components/post_header/index.js

```diff
@@ -23,7 +23,7 @@ function makeMapStateToProps() {
         const militaryTime = getBool(state, Preferences.CATEGORY_DISPLAY_SETTINGS, 'use_military_time');
 
         return {
-            commentedOnDisplayName: displayUsername(commentedOnUser, teammateNameDisplay),
```

Comment:
I think we should make this conditional on `ownProps.commentedOnUserId`. We still want this to show "Commented on Someone's message..." if we haven't loaded the user properly.